shared_infra/iam_policy_document.tf

```diff
@@ -1,3 +1,20 @@
+data "aws_iam_policy_document" "wellcomecollection-miro-images-public" {
+  statement {
+    actions = [
+      "s3:GetObject",
+    ]
+
+    resources = [
+      "arn:aws:s3:::${aws_s3_bucket.wellcomecollection-miro-images-public.id}/*",
+    ]
+
+    principals {
```

Comment:
Do we need this `principals` block? Is it used anywhere else?

Reply:
This is a bucket policy, so it's a bit different from the others policy documents. The principals block here is granting everyone permission to read any object.

If you look at the `alb_logs` policy it also has a principals block which grants permission to our account to put logs in an S3 bucket (basically giving our account permission to record ALB logs).

Reply:
👍